### PR TITLE
publish kiwi-ng based beta installer howto. Fixes #243

### LINKED ISSUE #243: publish kiwi-ng based beta installer howto
As part of our in-process transition to the new 'Built on openSUSE' relaunch associated with the new Rockstor 4 major version jump we have a new, as yet unreleased, kiwi-ng created installer. As our ultimate aim of distributing the pre-built installer itself depends upon complying with various trademark holders pre-requisites we must first provide details of what it is we are proposing to distribute. This issue is to address one element of this setup by first publishing the user orientated installer howto that further serves as an indication of what we are proposing to publish in binary form.

This howto, which is likely a 'first contact' user document, also serves to clarify our relationship with upstream. I.e. that we are not intending to be releasing anything that is claiming to be 'openSUSE' but clearly, as per the provided guidelines "Built on openSUSE"; this is an important element of our above referenced 'setup'.

References:
Kiwi-ng: https://osinside.github.io/kiwi/
openSUSE:Trademark guidelines: https://en.opensuse.org/openSUSE:Trademark_guidelines

### PR DESCRIPTION
**Pull request by phillxnet**
Adds the indicated howto along with a link within the existing howtos page and includes all associated images. The existing reinstall howto was also trivially updated to include a reference to the new installer and this howto.

Fixes #243
Please see issue text for context.

Ready for review.

All added links were tested and formatting was as expected in both Chrome and Firefox.

It is proposed that this howto is updated to link to the Rockstor kiwi-config repository once this has been published. We can also later add more prominent links to this howto in our getting started section. But given this installer is as yet not published it would seem not yet appropriate to do so.

### Checklist
- [x] With the proposed changes no Sphinx errors or warnings are generated.
- [x] I have added my name to the AUTHORS file, if required (descending alphabetical order).

### CODE REVIEW

File: installer-howto/installer-howto.rst [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+
+Thanks
+------
+
+Rockstor’s developers would like to thank the entire OSS community for making any of this possible.
+Specifically with our fledgling move / rebase on openSUSE Leap15.2 our thanks go to the openSUSE/SuSE organisations
+and the larger community for the extreme generosity we all benefit from in a myriad of seen and unseen ways.
+
+**Please note: Rockstor is in no way affiliated with openSUSE / SuSE and does not wish, under any circumstances, to misrepresent these organisations.**
+*Any errors you encounter are most likely our fault and we try, on a current volunteer basis, to support our users via our* `friendly forum: <https://forum.rockstor.com/.>`_.
```
Comment (FroggyFlox) [inline installer-howto/installer-howto.rst:16]: It seems you intended on making the url of the forum displayed, but it isn't in my case. I thus see a trailing ":" that is undesirable. We may want to remove that ":".

File: installer-howto/installer-howto.rst [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+and if this is so for you please consider either subscribing to our Stable Channel updates or taking part in our developer orientated Testing Channel.
+Details of these Rockstor package update services are available in our `Official Docs <http://rockstor.com/docs/>`_ subsection `Update Channels <http://rockstor.com/docs/update-channels/update_channels.html>`_.
+Rockstor’s ongoing development is wholly dependent on community support via either of these channels and by positively participating in our forum, linked above.
+It is hoped that in time we can contribute back more significantly, and possibly financially, as our own community and Stable Channel subscriptions grow.
+Also note that irrespective of Rockstor update channel selection or otherwise, upstream updates are available by normal command line means and via our Web-UI.
+See :ref:`dash_and_updates` later in this guide for the details.
+
+.. _our_kiwi_ng_installer:
+
+Our "Built on openSUSE" Wiki-ng created installer
```
Comment (FroggyFlox) [inline installer-howto/installer-howto.rst:28]:
```suggestion
Our "Built on openSUSE" Kiwi-ng created installer
```

File: installer-howto/installer-howto.rst [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+
+.. _dash_and_updates:
+
+
+Dashboard and System updates
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As the GPLv2 + licensed "rockstor" package stands on the shoulders of numerous OSS giants,
+it is possible to update all upstream, read openSUSE/SuSE, provided/curated packages
+via the flashing ico to the left of the "Uses openSUSE ..." text in the top right of the Web-UI:
```
Comment (FroggyFlox) [inline installer-howto/installer-howto.rst:213]:
```suggestion
via the flashing icon to the left of the "Uses openSUSE ..." text in the top right of the Web-UI:
```

File: installer-howto/installer-howto.rst (hunk 2 above)
Comment (phillxnet) [reply installer-howto/installer-howto.rst:28]: @FroggyFlox Thanks, you wouldn't believe the number of times I keep doing this.

File: installer-howto/installer-howto.rst (hunk 1 above)
Comment (phillxnet) [reply installer-howto/installer-howto.rst]: @FroggyFlox I think we are OK, within our 'official docs' with not showing the URL, especially when it's to our own domain.  This was, as you suspected, just a stray colon as I was battling a url embedded in an italic block. We use this same "friendly forum" / "communit forum" link elsewhere in the docs and on the main web page. So I've, as suggested pushed a commit (3b77831) to just remove the stray colon.

Thanks for catching this. Much appreciated. We could probably draw up some guidelines for when to show url as link text.

**Comment by phillxnet** [comment]
@FroggyFlox Thanks for the review, and fancy me submitting that wiki / kiwi. Never mind.
I've now made all suggested changes.

I'll likely publish this along side the actual kiwi-config repo so when folks enquire about this as yet unreleased in binary form installer we have somewhere to point them while we make the necessary steps prior to setting up a public download.

**Pull request merged**